### PR TITLE
Add articles about vscode-xml

### PR DESCRIPTION
**Pull request by angelozerr**
Add articles about vscode-xml

Signed-off-by: azerr <azerr@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,6 +84,12 @@ The following settings are supported:
 Since 0.13.0:
 * `files.trimTrailingWhitespace`: Now affects XML formatting. Enable/disable trailing whitespace trimming when formatting an XML document. Default is `false`.
 
+## Articles
+
+ * [What’s new in the Visual Studio Code XML Extension 0.8.0](https://developers.redhat.com/blog/2019/07/24/whats-new-in-the-visual-studio-code-xml-extension/)
+ * [Eclipse LemMinX version 0.11.1](https://developers.redhat.com/blog/2020/03/27/red-hat-xml-language-server-becomes-lemminx-bringing-new-release-and-updated-vs-code-xml-extension/)
+ * [Improved schema binding and more in Red Hat XML extension for VS Code 0.12.0 and LemMinX](https://developers.redhat.com/blog/2020/07/02/improved-schema-binding-and-more-in-red-hat-xml-extension-for-vs-code-0-12-0-and-lemminx/)
```
Comment (xorye) [inline README.md:91]: On second thought, maybe we should have the more recent articles at the top? If yes, I can make that change on the vscode-quarkus readme https://github.com/redhat-developer/vscode-quarkus too.

Comment (angelozerr) [reply README.md:91]: My idea is to sort articles per new features, but if @fbricon think it's better to change the order, I will do that.

Comment (fbricon) [reply README.md:91]: +1 for more recent articles on top

Comment (angelozerr) [reply README.md:91]: Please see my PR https://github.com/redhat-developer/vscode-xml/pull/286 which fixes that.

**Comment by angelozerr** [comment]
Duplicate of https://github.com/redhat-developer/vscode-xml/pull/286

**Pull request closed without merging**